Hmm.  Old version compiled file, but now I get:

```
g++ -c -m64 -pipe -std=c++0x -O2 -fno-strict-aliasing -pthread -fPIC -Wall -W -D_REENTRANT -fPIE -DPQCONSOLE_STATIC -DPL_SAFE_ARG_MACROS -DPQCONSOLE_NO_HTML -DPQCONSOLE_BROWSER -DPQCONSOLE_HANDLE_HOOVERING -DLQUTY_STATIC -DQT_NO_DEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_CORE_LIB -I/usr/lib/x86_64-linux-gnu/qt5/mkspecs/linux-g++-64 -I. -I../../../../lib/swipl/include -I/usr/include/qt5 -I/usr/include/qt5/QtWidgets -I/usr/include/qt5/QtGui -I/usr/include/qt5/QtCore -I. -o main.o main.cpp
main.cpp: In function ‘void logger(QtMsgType, const QMessageLogContext&, const QString&)’:
main.cpp:88:10: error: ‘QtInfoMsg’ was not declared in this scope
     case QtInfoMsg:
          ^
make: *** [main.o] Error 1
```

P.s.     Normally I'm the one closing a pull request after merging or the submitter closes it after discovering the request was not a good idea.

--- a/main.cpp
+++ b/main.cpp
@@ -85,9 +85,11 @@ static void logger(QtMsgType type, const QMessageLogContext &context, const QStr
         fprintf(logfile, "Fatal: %s (%s:%u, %s)\n", localMsg.constData(), context.file, context.line, context.function);
         abort();
         break;
+#if QT_VERSION >= 0x050500
     case QtInfoMsg:
         fprintf(logfile, "Info: %s (%s:%u, %s)\n", localMsg.constData(), context.file, context.line, context.function);
         break;
+#endif
     }
 
     fflush(logfile);

